src/test/js/notMultibranch/freestyleRunning.js

```diff
@@ -0,0 +1,27 @@
+/** @module freestyleRunning
```

Comment:
@michaelneale you should add an index file for the documentation and then refer to that index as @memberof :ant: